test/services/PackageTypesSpec.scala

```diff
@@ -16,21 +16,24 @@
 
 package services
 
+import services.model.PackageType
 import uk.gov.hmrc.play.test.UnitSpec
 
-class PackageTypeSpec extends UnitSpec {
+class PackageTypesSpec extends UnitSpec {
 
   "Package type list" should {
+    
     "return package types containing commas and quotes" in {
-      val somePackageTypes: List[PackageType] =
-        PackageType.all.filter((packageType: PackageType) => packageType.code == "43")
+      val somePackageTypes = PackageTypes.all.filter(_.code == "43")
```

Comment:
👍

Reply:
My bad, I did this for Nicks benefit, forgot to remove